docs/src/content/docs/zh-cn/reference/configuration.mdx

```diff
@@ -444,15 +444,30 @@ Starlight 默认使用 Sarah Drasner 的 [Night Owl 主题](https://github.com/s
 
 ### `pagefind`
 
-**类型：** `boolean`  
+**类型：** <code>boolean | <a href="#pagefindoptions">PagefindOptions</a></code> 
```

Comment:
```suggestion
**类型：** <code>boolean | <a href="#pagefindoptions">PagefindOptions</a></code>  
```

The trailing double space got removed in the translation process, so the type and default value are now on the same line instead of 2:

<img width="398" alt="image" src="https://github.com/user-attachments/assets/037b9d52-627e-42de-8390-80bf4ffa286d" />